packages/components/nodes/vectorstores/OpenSearch/OpenSearch.ts

```diff
@@ -3,8 +3,9 @@ import { Client } from '@opensearch-project/opensearch'
 import { Document } from '@langchain/core/documents'
```

Comment:
curious: why the need to move the URL into credential?

Reply:
Hi... when we are using OpenSearch with username and password, we need to pass credentials through URL. This situation is similar to MongoDb Atlas connection string. Because this I think OpenSearch URL as a connection string and should be a credential.

Reply:
got it make sense, thanks! Also can we increment the version number of the node

Reply:
Good. I will fix some errors I found.